crates/augurs-outlier/src/dbscan.rs

```diff
@@ -82,6 +82,17 @@ impl DbscanDetector {
         })
     }
 
+    /// Set epsilon for the DBSCAN algorithm.
+    pub fn set_epsilon(mut self, epsilon: f64) {
+        self.epsilon_or_sensitivity = EpsilonOrSensitivity::Epsilon(epsilon);
+    }
+
+    /// Set sensitivity for the DBSCAN algorithm.
+    pub fn set_sensitivity(mut self, sensitivity: f64) {
+        self.epsilon_or_sensitivity =
+            EpsilonOrSensitivity::Sensitivity(Sensitivity::try_from(sensitivity).unwrap());
```

Comment:
We shouldn't unwrap here; we should return a `Result<(), Error>` instead.
```suggestion
            EpsilonOrSensitivity::Sensitivity(Sensitivity::try_from(sensitivity)?);
            Ok(())
```